css/properties/column-gap.json

```diff
@@ -62,26 +65,201 @@
+            ]
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "percentage_values": {
+          "__compat": {
+            "description": "<code>&lt;percentage&gt;</code> values",
+            "support": {
+              "chrome": {
+                "version_added": "66"
+              },
+              "chrome_android": {
+                "version_added": "66"
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": "16"
+              },
+              "firefox": {
+                "version_added": "61"
+              },
+              "firefox_android": {
+                "version_added": "61"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": "66"
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "calc_values": {
+          "__compat": {
+            "description": "<code>calc()</code> values",
+            "support": {
+              "chrome": {
+                "version_added": "66"
+              },
+              "chrome_android": {
+                "version_added": "66"
+              },
+              "edge": {
+                "version_added": "16"
+              },
+              "edge_mobile": {
+                "version_added": "16"
+              },
+              "firefox": {
+                "version_added": "61"
+              },
+              "firefox_android": {
+                "version_added": "61"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": "66"
+              }
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "grid_and_flex_box": {
+          "__compat": {
+            "description": "Support for <a href='https://developer.mozilla.org/ddocs/Web/CSS/CSS_Grid_Layout'>grid</a> and <a href='https://developer.mozilla.org/docs/Web/CSS/CSS_Flexible_Box_Layout'>flexible box</a> layouts",
```

Comment:
Giving this one last look over and there's a typo in the first URL here (`ddocs`)

Reply:
I’ve fixed it now, but the linter should probably catch stuff like this.

Reply:
Yeah, there's a bit of a history around this problem—see #1998 and the other issues mentioned there. I have also written a pre-commit hook to try to catch these in my changes; maybe it could be generalized. I'll open an issue to try to address this.